staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -84,7 +91,14 @@ func (a *AdmissionOptions) AddFlags(fs *pflag.FlagSet) {
 		"The names in the below list may represent a validating plugin, a mutating plugin, or both. "+
 		"Within each phase, the plugins will run in the order in which they are passed to this flag. "+
 		"Comma-delimited list of: "+strings.Join(a.Plugins.Registered(), ", ")+".")
-
+	fs.MarkDeprecated("admission-control", "Use --enable-admission-plugin or --disable-admission-plugin instead. Will be removed in a future version.")
+	fs.StringSliceVar(&a.EnableAdmissionPlugins, "enable-admission-plugin", a.EnableAdmissionPlugins, ""+
```

Comment:
enable-admission-plugin or enable-admission-plugins? I tend to the later.

Reply:
ok